src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -618,6 +618,35 @@ public Object write(TruffleObject receiver, Object identifier, Object value,
 
     }
 
+    @CoreMethod(names = "remove", isModuleFunction = true, required = 2)
+    @ImportStatic(Message.class)
+    public abstract static class RemoveNode extends CoreMethodArrayArgumentsNode {
+
+        @Specialization
+        public boolean remove(TruffleObject receiver, Object identifier,
+                @Cached("create()") RubyToForeignNode identifierToForeignNode,
+                @Cached("REMOVE.createNode()") Node removeNode,
+                @Cached("create()") BranchProfile unknownIdentifierProfile,
+                @Cached("create()") BranchProfile exceptionProfile) {
+
```

Comment:
Sorry, random blank line here, and actually an extra blank line after the `RemoveClass` as well.

Reply:
I made it look like the "read" and "write" messages directly above it.